src/extension.ts

```diff
@@ -708,6 +712,22 @@ export function activate(context: vscode.ExtensionContext) {
       }
     )
   );
+
+  subscriptions.push(
+    vscode.commands.registerCommand(
+      "vscode-couchbase.openQueryNotebook",
+      async () => {
+        createNotebook();
```

Comment:
Very good! This file is already quite big, the more we break it into smaller pieces, the better.